### PR TITLE
Auto Commenting for Issue Raised

### PR DESCRIPTION
**Pull request by Rajneeshkarya**
**Issue Fixed**

Auto-Commenting Feature for Issue Raised #138 

**Description**

This pull request introduces a new GitHub Actions workflow that automatically comments on new and updated issues. This feature aims to streamline the contribution process by providing contributors with immediate feedback and guidance upon creating or updating their issues.

**Screenshots**

Screenshot_2024-05-17_22_50_09

### CODE REVIEW

**Comment by Rajneeshkarya** [comment]
Hi @jfmartinz,

I've added the auto-comment feature for the Issue raised as well

**Comment by jfmartinz** [comment]
Please remove the "[Auto Comment:]". Also I have problem with the PR merged, not working. I dk whats the problem haha, I need your idea on this why its not working.

**Comment by jfmartinz** [comment]
I updated the yaml file

**Comment by Rajneeshkarya** [comment]
hi @jfmartinz 

You can merge these commits, this is for Issue Raised only, I will create an issue for the PR merged then we can work on that.

File: .github/workflows/autocomment-pr-raise.yml [hunk 1]
```diff
@@ -34,3 +37,30 @@ jobs:
...
+  comment-issue:
+    runs-on: ubuntu-latest
+    if: github.event_name == 'issues'
+
+    steps:
+    - name: Add Comment to Issue
+      run: |
+        COMMENT=$(cat <<EOF
+        {
+          "body": "[Auto-Comment]: Thank you for raising this issue! 🚀 We've received your input and will review it shortly. Please make sure to check our CONTRIBUTING.md for any additional information. Feel free to reach out if you have any questions or need further assistance. Happy contributing! 😊"
```
Comment (jfmartinz) [inline .github/workflows/autocomment-pr-raise.yml:50]:
```suggestion
          "body": "Thank you for raising this issue! 🚀 We've received your input and will review it shortly. Please make sure to check our CONTRIBUTING.md for any additional information. Feel free to reach out if you have any questions or need further assistance. Happy contributing! 😊"
```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Pull request closed without merging**